requirements.txt

```diff
@@ -30,7 +30,7 @@ git+https://github.com/mingchen/django-cas-ng@69463fe7c23f025be7165c2967f4ceb983
 # Importer application requirements
 xbundle==0.3.0
 git+https://github.com/gdub/python-archive.git@61b9afde21621a8acce964d3336a7fb5d2d07ca6#egg=python-archive==0.3.1-odl
-django-storages-redux==1.2.3
+django-storages-redux==1.3
```

Comment:
this looks good, drops older django support, and removed a bunch of deprecated stuff we weren't using.